lib/membrane/video_compositor/scene/object/input/static_frame.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Membrane.VideoCompositor.Scene.Object.Input.StaticFrame do
```

Comment:
A 'Static Frame' is just an image. I'm not 100% sure, but I would consider renaming this module to `Image`.